### PR TITLE
[CIVIS-3137] Replace wall time documentation with new metric "total sessions duration"

### PR DESCRIPTION
**Pull request by TovRudyy**
### What does this PR do? What is the motivation?

Remove from our documentation mentions to "wall time" and replace it with the new metric "total sessions duration".

JIRA task: [CIVIS-3137](https://datadoghq.atlassian.net/browse/CIVIS-3137)

### Merge instructions

- [X] Please merge after reviewing

### Additional notes

[CIVIS-3137]: https://datadoghq.atlassian.net/browse/CIVIS-3137?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/continuous_integration/guides/flaky_test_management.md
Comment (urseberry) [inline content/en/continuous_integration/guides/flaky_test_management.md:25]:
```suggestion
For each branch, the list shows the number of new flaky tests introduced by the commit, the number of flaky commits, total test time, and the branch's latest commit details. 
```

File: content/en/continuous_integration/search/_index.md
Comment (urseberry) [inline content/en/continuous_integration/search/_index.md:105]:
```suggestion
- View the total duration of your tests in your feature branch.
```

File: content/en/continuous_integration/search/_index.md
Comment (urseberry) [inline content/en/continuous_integration/search/_index.md:126]:
```suggestion
For each branch, you can see the test service, the number of failed, passed, and skipped tests, test regressions, total test time, when the commit was last updated, and the avatar of the author of the commit.
```

File: content/en/continuous_integration/search/_index.md
Comment (urseberry) [inline content/en/continuous_integration/search/_index.md:128]:
```suggestion
Hovering over the commit author's avatar shows detailed information about the latest commit.
```

File: content/en/continuous_integration/troubleshooting.md
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:96]:
```suggestion
### The total test time is empty
```

File: content/en/continuous_integration/troubleshooting.md
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:97]:
```suggestion
If you cannot see the total test time, it is likely that test suite level visibility is not enabled. To confirm, check if your language supports test suite level visibility in [Supported features][15]. If test suite level visibility is supported, update your tracer to the latest version.
```

File: content/en/continuous_integration/troubleshooting.md
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:101]:
```suggestion
### The total test time is different than expected
```

File: content/en/continuous_integration/troubleshooting.md
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:104]:
```suggestion
The total time is defined as the sum of the maximum test session durations.
```

**Comment by TovRudyy** [comment]
Thank you @urseberry for the review 🙌

**Review by urseberry — APPROVED** [review]

**Comment by urseberry** [comment]
Is this ready to merge? If so, please check the "please merge after reviewing" checkbox in the description.

**Comment by TovRudyy** [comment]
> Is this ready to merge? If so, please check the "please merge after reviewing" checkbox in the description.

No. I plan to merge this the 23rd of November because it is blocked by a small backend change.

**Pull request merged**
